Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -149,6 +149,34 @@ public sealed class StyleNano : StyleBase
 
         public static readonly Color ChatBackgroundColor = Color.FromHex("#25252ADD");
 
+        // DeltaV - AAC button variables
+        public static readonly string CommandButtonClass = "CommandButton";
```

Comment:
upstream is likely to nuke this file in the future so put it elsewhere

Reply:
where should i put it?

Reply:
make a file called DeltaVStyle.cs or something so it will never have conflicts